### ISSUE TITLE
titlecase caps #hash 

### ISSUE THREAD

**Issue by CarlFK** [chrisdrackett/django-typogrify#16]
I'm not really sure what 'right' is, but in one case it hosed me, and my gut says leave words with prefix chars alone.
```
>>> titlecase( '#hash #tag')
'#Hash #Tag'
```

**Comment by chrisdrackett** [comment]
I would agree here, pull requests welcome! (I don't work in python anymore and have not been able to work on this library in a long time)
